addons/website/views/snippets/s_carousel_intro.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_carousel_intro" name="Carousel Intro">
+    <section class="s_carousel_intro_wrapper p-0">
+        <t t-set="uniq" t-value="datetime.datetime.now().microsecond"/>
+        <div t-attf-id="myCarouselIntro{{uniq}}" class="s_carousel_intro s_carousel_default carousel slide" data-bs-interval="10000">
+            <!-- Content -->
+            <div class="carousel-inner">
+                <!-- #01 -->
+                <div class="carousel-item active o_cc o_cc5 oe_img_bg o_bg_img_center px-0 pt152 pb152" style="background-image: url('/web/image/website.s_carousel_intro_default_image_1');" data-name="Slide">
+                    <div class="o_we_bg_filter bg-black-50"/>
```

Comment:
This was made on purpose to handle industry images. Since we don't know which kind of images the Web Builder will define if the user add this snippet, this will ensure the text will always be visible by default.
Or maybe using a white filter ? Mhh what do you think ? 🤔

Reply:
> This was made on purpose to handle industry images.

Right. We should then change the demo images I'm afraid... let's use others that suit better an overlay 💪

Reply:
Ok, I updated the images 👍